cypress/integration/pages/articles/tests.js

```diff
@@ -145,6 +148,26 @@ export const testsThatFollowSmokeTestConfig = ({
         );
       });
 
+      if (
+        serviceHasInlineLink(service) &&
```

Comment:
Isn't it simpler to just do `service ==='news'`? Instead of creating a function we only use once.

Reply:
yeah but i just followed the previous code of creating `const` and using in the code below